.gitlab/custom-jobs-and-variables.yml

```diff
@@ -56,5 +68,6 @@ variables:
 # CI behavior.
 # See Umpire for an example (export junit test reports).
 .custom_job:
-  variables:
-    JOB_TEMPLATE_CANNOT_BE_EMPTY: "True"
+  artifacts:
```

Comment:
I copied this from RAJA and Umpire.

Reply:
✅ You can indeed export these tests reports for CHAI too, since the build_and_test script is producing them.